docs/_sass/_style.scss

```diff
@@ -1031,3 +1031,11 @@ code.output {
   clip: rect(0, 0, 0, 0);
   border: 0;
 }
+
+.result {
+  border: 1px solid yellow;
```

Comment:
updated, but i did leave the padding style b/c without it, the text is right against the line.